src/pages/OnBoarding/util/OnBoardingContext.jsx

```diff
@@ -118,9 +118,9 @@ export const OnBoardingProvider = ({ children, initStep, onFinish }) => {
   const [selectedAddons, setSelectedAddons] = useState([])
 
   // get selected release data
-  const { data: release = {} } = useGetReleaseQuery(
+  const { data: release = {}, isLoading: isLoadingAddons } = useGetReleaseQuery(
     { name: selectedPreset },
-    { skip: !selectedPreset || stepIndex !== 7 },
+    { skip: !selectedPreset || stepIndex < 6 },
```

Comment:
After testing I think this should be `< 5`. When going to step 6 we have to wait for the release to load the data. It's smoother if we pre-fetch the release on step 5 so the user doesn't have to wait on step 6.

Reply:
Moved to step 5.